chef/cookbooks/crowbar/recipes/crowbar-upgrade.rb

```diff
@@ -31,6 +31,18 @@
 
   include_recipe "crowbar::stop-services-before-upgrade"
 
+  # Stop rabbitmq
+  bash "stop rabbitmq resources" do
+    code <<-EOF
+        for resource in g-rabbitmq rabbitmq fs-rabbitmq ms-drbd-rabbitmq drbd-rabbitmq;
+        do
+          crm --force --wait resource stop $resource
+        done
+      done
+    EOF
+    only_if { ::File.exist?("/usr/sbin/crm") }
+  end
```

Comment:
I am not completely sure but I think it is possible to use execute blocks here instead similar to how it is done here https://github.com/crowbar/crowbar-core/pull/301